.github/workflows/publish-release.yml

```diff
@@ -1,40 +1,25 @@
-name: Publish Bundle
-
+---
+name: publish-release
 on:
   push:
-    tags:
-      - v*
-
+    tags: ['v*']
 jobs:
   build:
-    runs-on: ubuntu-latest
-
+    runs-on: ubuntu-22.04
```

Comment:
Curious why we set this to a specific version

Reply:
Probably for stability. But we should match whatever netlify uses. Do we know what they use?